identifier/src/containers/Login/Login.jsx

```diff
@@ -134,14 +144,28 @@ function Login(props) {
           className={classes.usernameInputField}
         />
         <TextField
-          type="password"
+          type={showPassword ? "text" : "password"}
           label={t("konnect.login.passwordField.label", "Password")}
           error={!!errors.password}
           helperText={<ErrorMessage error={errors.password}></ErrorMessage>}
           fullWidth
           onChange={handleChange('password')}
           autoComplete="kopano-account current-password"
           variant="outlined"
+          InputProps={{      // Added a end adornment of Visibility Icon to show/hide password entered in the field.
```

Comment:
Also here, please use imperative mood. In this case something like `// End-adornment icon to show/hide password.`

Reply:
Updated the comment in the commit itself.